internal/kubernetes/wait.go

```diff
@@ -10,7 +10,10 @@ import (
 	"github.com/aws/eks-hybrid/internal/retry"
 )
 
-const minWait = time.Duration(200 * time.Millisecond)
+const (
+	minWait = time.Duration(200 * time.Millisecond)
```

Comment:
nit
this was "wrong" before
```suggestion
	minWait = 200 * time.Millisecond
```